### DISCUSSION TITLE
Add bypass status sensor

### DISCUSSION THREAD

**Discussion by Takethis88** [veista/nilan#97] [Ideas]
Hi Viesta. Thank you for your integration. 

I can't see the status of the bypass on my system. VP18cCom modbus v9
I'm trying to troubleshoot what I believe to be an anomalous or in the least annoying behaviour by the MVHR system and knowing if the bypass is open or close at a glance would be useful.

Also knowing what kind of heating type is being used would be great to try maximise the efficiency of the system (hp, electrical, electric rele..)

**Comment by veista** [comment]
Hi,

Thanks for bringing this up. I will look into this. I believe the bypass is left out by mistake, but if I remember correctly the heating type is not supported in bus version 9.

**Reply by veista** [reply]
Btw, you can easily enable features in the device_map.py and check if they work on your device.

**Reply by veista** [reply]
I checked, bypass flap state also requires bus version 11, but it is also missing from the device map for your device.

**Comment by Takethis88** [comment]
I see. Thank you for checking. I've tried manually editing the device map
but the sensor doesn't return any result. Since the device isn't on the
map, does it mean that even if I upgrade the firmware it won't be supported?

**Reply by veista** [reply]
After you upgrade the firmware to bus 19, everything your device has to offer will be opened. It still requires for me (or you) to add it to the device map.
